WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -123,15 +124,15 @@ public class AppSettingsViewController: UITableViewController {
             } else {
                 WPAnalytics.track(.EditorToggledOff)
             }
-            WPPostViewController.setNewEditorEnabled(enabled)
+            EditorSettings().visualEditorEnabled = enabled
```

Comment:
Switching to a class property instead of instantiating `EditorSettings ` directly would make `AppSettingsViewController.swift` slightly friendlier for unit-testing, since we could use dependency injection (through a custom initializer) to replace the editor settings object.